docs/build/get-started.md

```diff
@@ -751,9 +751,29 @@ That means your Deal was deployed successfully. After a short while, Providers w
 
 ### Call a function over the subnet
 
-In your freshly-initialized project, there's a file `src/aqua/main.aqua` which contains the function `runDeployedServices` which resolves a Subnet and calls a function on each of the Workers in the Subnet.
+In your project, there's a file `src/aqua/main.aqua` which contains the function `runDeployedServices` which resolves a Subnet and calls a function on each of the Workers in the Subnet.
 
-To run it, simply run the following command:
+However, since we have started with a `minimal` template, this function is commented.
+
+Uncomment it, and then change the service function call that's written to `answer` variable.
+
+Change these two lines:
+```aqua
+answer <- MyService.greeting("fluence")
+answers <<- Answer(answer=?[answer], worker=w)
+```
+
+To this
+```aqua
+answer <- HelloWorld.hello_world()
+answers <<- Answer(answer=?[answer.response], worker=w)
+```
+
+Let's go through the two changes:
```

Comment:
Oh, additionally thank you for the kinda new valuable info. Now it seems to be workable and consistent